salesagility/SuiteCRM
# Fixed #5902: Not audited if the user was not update for the value of the field type 'text', 'varchar'

## Description
See detail in #5902 




## Motivation and Context

The system has audited the value incorrect for some fields

## How To Test This

We are 2 case need verify:
Case 1:
1. Go to any module > Create new record > In Name field type value contain character double quotes (").
2. From detail record > Click on 'Action button' > Edit > Do not change any thing > Save
3. View change Log

Expected:
- Name field should not audited because The user has not any change

Case 2: (Technical test, need a developer test to make sure)
1. Go to studio and add field type of 'text', MUST integrate TinyMCE editor plugin for this field.
2. Create new record on module just added field at step 1 (Must type value for text field created in step 1) > Save
3. From record detail > Click on Action button > Edit > Do not change any thing > Save.
4. View change log

Expected:
- Text field created in step 1 should not audited because The user has not any change

## Types of changes


### Final checklist
